README.rst

```diff
@@ -491,7 +491,7 @@ Server and create required Scopes.
             library will not be able to, and will therefore not attempt to, retrieve
             a user's groups and so no policies based on Groups may be used. We
             encourage you to consult the `Globus Auth Documentation
-            <https://docs.globus.org/api/auth/>`_ for more information on creation
+            <https://docs.globus.org/api/auth/>`__ for more information on creation
```

Comment:
Should this be a double underscore? Perhaps that's RST I don't know off the top of my head.

Reply:
Yea the rst linter I'm running caught that as a `Duplicate explicit target name error`. Apparently, using a double underscore marks the link as an anonymous reference (seen from this [stack overflow q](https://github.com/sphinx-doc/sphinx/issues/3921#issuecomment-315581557))

Reply:
that's new to me. What rst linter are you using? Perhaps I should do it also (and maybe we should make it a test target of some sort).